lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -156,8 +157,8 @@
     only_if "git status --porcelain |grep \".\""
   end
 
-  execute("git-return-to-master-branch") do
-    command("git checkout master")
+  execute("git-return-to-#{pipeline}-branch") do
+    command("git checkout #{pipeline}")
```

Comment:
No, `add-delivery-configuration` is the branch we want to merge into whatever pipeline is. `add-delivery-configuration` is a temporary branch created by this recipe to merge some changes related to the delivery cookbook into `pipeline`.